internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -164,3 +365,42 @@ func stackExists(err error) bool {
 	}
 	return false
 }
+
+// stackExists returns true if the underlying error is a stack already exists error.
+func stackSetExists(err error) bool {
+	if aerr, ok := err.(awserr.Error); ok {
+		switch aerr.Code() {
+		case cloudformation.ErrCodeNameAlreadyExistsException:
+			// An ErrCodeNameAlreadyExistsException occurs when a stack set already exists.
+			return true
+		}
+	}
+	return false
+}
+
+func (cf CloudFormation) waitForStackSetOperation(stackSetName, operationID string) error {
+	for {
+		response, err := cf.client.DescribeStackSetOperation(&cloudformation.DescribeStackSetOperationInput{
+			OperationId:  aws.String(operationID),
+			StackSetName: aws.String(stackSetName),
+		})
+
+		if err != nil {
+			return err
+		}
```

Comment:
Can we wrap this error :) with the name and operation id

There a bunch of errors from public methods that are not wrapped. I'm adding the comment only here :)

Reply:
<3 Yeap yeap. I was going to fix this on Sunday but ended up watching "The Good Place" all day.